### PR TITLE
Cards vue refactor - description(1)

### PR DESCRIPTION
**Pull request by chosta**
Adding `effect` and `action` instead of effectBox + streamlining description for these elements. Also a couple of cards after the change.

### CODE REVIEW

File: src/cards/render/CardRenderer.ts [hunk 1]
```diff
@@ -83,9 +83,12 @@ export class CardRenderEffect extends CardRenderer {
 
   public get description(): ItemType {
     this._validate();
-    // TODO (chosta): validate builder method to make sure it's the last element
```
Comment (chosta) [inline src/cards/render/CardRenderer.ts:86]: This is kind of not needed now as setting a description is pushing the string to the last row (see `public set description`) and there is already a check if effectBox has a cause, effect and description

**Comment by kberg** [comment]
Note conflict on four files.

File: src/cards/base/AquiferPumping.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ export class AquiferPumping extends Card implements IActionCard, IProjectCard {
       metadata: {
         cardNumber: '187',
         renderData: CardRenderer.builder((b) => {
-          b.effectBox((eb) => eb.megacredits(8).steel(1).brackets.startAction.oceans(1).description('Action: Spend 8 MC to place 1 ocean tile. STEEL MAY BE USED as if you were playing a Building card.'));
+          b.action('Spend 8 MC to place 1 ocean tile. STEEL MAY BE USED as if you were playing a Building card.', (eb) => eb.megacredits(8).steel(1).brackets.startAction.oceans(1));
```
Comment (kberg) [inline src/cards/base/AquiferPumping.ts:27]: nit: linebreak? Very minor.

File: src/cards/base/BusinessNetwork.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ export class BusinessNetwork extends Card implements IActionCard, IProjectCard {
         cardNumber: '110',
         description: 'Decrease your MC production 1 step.',
         renderData: CardRenderer.builder((b) => {
-          b.effectBox((eb) => eb.empty().startAction.empty().description()).text('Action: Look at the top card and either buy it or discard it', CardRenderItemSize.SMALL, true).br;
+          b.action(undefined, (eb) => eb.empty().startAction.empty()).text('Action: Look at the top card and either buy it or discard it', CardRenderItemSize.SMALL, true).br;
```
Comment (kberg) [inline src/cards/base/BusinessNetwork.ts:29]: Why is text necessary if description is the first parameter to action()?

File: src/cards/render/CardRenderer.ts [hunk 4]
```diff
@@ -441,6 +444,27 @@ class Builder {
     return this;
   }
 
+  public standartProject(description: string, eb: (builder: EffectBuilder) => void): Builder {
```
Comment (kberg) [inline src/cards/render/CardRenderer.ts:447]: "standart"

**Review by kberg — APPROVED** [review]

**Comment by kberg** [comment]
I love it.

File: src/cards/base/BusinessNetwork.ts (hunk 3 above)
Comment (chosta) [reply src/cards/base/BusinessNetwork.ts:29]: It's a hack. The original iconography requires bolded uppercase text, while description anywhere else is teeny-tiny non-formatted text. I'd just roll with it for now.

File: src/cards/render/CardRenderer.ts (hunk 4 above)
Comment (chosta) [reply src/cards/render/CardRenderer.ts:447]: I'll probably keep making this mistake as this word ends in `t` in my language. I need to find a way to hack my brain to stop it!

**Pull request merged**